Materialized system view paths docs description

n00bcracker opened the pull request:
### Changelog entry <!-- a user-readable short description of the changes that goes to CHANGELOG.md and Release Notes -->

Materialized system view paths documentation for stable 25-4

### Changelog category <!-- remove all except one -->

* Documentation (changelog entry is not required)

### Description for reviewers <!-- (optional) description for those who read this PR -->

...

n00bcracker committed:
Materialized system views docs description

n00bcracker committed:
Update system-views.md

n00bcracker committed:
Update system-views.md

n00bcracker committed:
Update system-views.md

azevaykin commented on ydb/docs/ru/core/devops/observability/system-views.md:
```diff
@@ -12,6 +12,29 @@
 
+{% note warning %}
+
+Начиная с версии 25-4 появилась возможность настраивать права доступа к каталогу `.sys` c системными представлениям кластера и отдельных баз, а также к самим системным представлениям.
```
Версионирование документации ведется в ветках, поэтому упоминания версий в тексте не нужно.
Сначала документация появляется в main, потом вольется в 25-4.
Поэтому этот абзац лучше убрать.

azevaykin replied:
Мне нравится, что есть упоминание про миграцию, но не нравится текущее расположение, это замусоривает изложение.
@orange13 куда лучше добавить?

n00bcracker committed:
Update glossary.md

n00bcracker committed:
Update system-views.md

azevaykin commented:
@orange13 
Я ок, с учетом вопросов к тебе выше.
https://github.com/ydb-platform/ydb/pull/29410#discussion_r2567599649
https://github.com/ydb-platform/ydb/pull/29410#discussion_r2567592759

orange13 committed:
review fixes

n00bcracker committed:
Update system views documentation by removing warning

Removed warning note about access rights to the .sys directory and system views.
